### PR TITLE
Publishes the AWS Lightsail deployment guide for the MML Starter Project

### PR DESCRIPTION
**Pull request by TheCodeTherapy**
This PR aims to publish the AWS Lightsail deployment guide for the MML Starter Project

**What kind of change does your PR introduce?** (check at least one)

- [ ] Bugfix
- [X] Feature
- [ ] Refactor
- [ ] Tests
- [ ] Other, please describe:

**Does your PR fulfill the following requirements?**

- [X] All tests are passing
- [X] The title references the corresponding issue # (if relevant)

### CODE REVIEW

**Review by sachamorgese — APPROVED** [review]
LGTM!

File: src/content/guides/mml-starter-project-hosting/body.mdx [hunk 1]
```diff
@@ -112,3 +112,128 @@ Similarly to CodeSandbox, Glitch allows you to “remix” the MML Starter Proje
 + Automatic re-deployments from GitHub
 - You can't share the code with others
 - You can’t easily modify the MML document live
+
+# AWS Lightsail
+
+* *Disclaimer: Deploying the MML starter project in AWS Lightsail or any other self-managed VPS service has a lot of complexity involved, so please keep in mind this is **not** one of the Fast setup options, but a more permanent solution for those who may see usage for such permanency and full controll of the deployment environment.*
```
Comment (MarcusLongmuir) [inline src/content/guides/mml-starter-project-hosting/body.mdx:118]: typo: `controll`/`control`

File: src/content/guides/mml-starter-project-hosting/body.mdx [hunk 2]
```diff
@@ -112,3 +112,128 @@ Similarly to CodeSandbox, Glitch allows you to “remix” the MML Starter Proje
...
+* Once you have your Instance created and running (it usually takes a few minutes after its creation), you can jump straight into accessing it through SSH using the little terminal icon on your Instance label. That will grant you access through SSH on a web browser. *(screenshot below)*
+![SSH through the Web BrowserS](/images/guides/aws-lightsail/lightsail-web-ssh.png)
+
+* In case you want to access it through SSH on a Terminal or other SSH applications like PuTTY, you can follow the steps explained here: [Understanding SSH in Amazon Lightsail](https://lightsail.aws.amazon.com/ls/docs/en_us/articles/understanding-ssh-in-amazon-lightsail) to manage the SSH keys required to access your VPS.
+
+* Update your Linux distribution through the following command:
+```bash
+sudo apt udate && sudo apt upgrade
```
Comment (MarcusLongmuir) [inline src/content/guides/mml-starter-project-hosting/body.mdx:129]: typo: `udate`/`update`

File: src/content/guides/mml-starter-project-hosting/body.mdx [hunk 3]
```diff
@@ -112,3 +112,128 @@ Similarly to CodeSandbox, Glitch allows you to “remix” the MML Starter Proje
...
+```
+
+* Run the project in development mode to check if everything runs fine:
+```bash
+npm run iterate
+```
+
+* There are a few extra steps so you can access your Starter Project. The first one is getting a static IP address for your LightSail Instance, which you can do by clicking a `Create static IP` button at the [Networking](https://lightsail.aws.amazon.com/ls/webapp/home/networking) tab.
+
+* The second one is to allow the connections to port 8080 (which is the port the Starter Project uses to receive incoming connections). To do so, you can do to your Instances tab, click on the three dots of your instance, click on Manage, go to the Networking tab, and add a new Custom Firewall rule to allow TCP connections to port `8080` *(please see the screenshots below)*.
```
Comment (MarcusLongmuir) [inline src/content/guides/mml-starter-project-hosting/body.mdx:161]: Add backticks to the 8080 in the first sentence to make it consistent

File: src/content/guides/mml-starter-project-hosting/body.mdx [hunk 4]
```diff
@@ -112,3 +112,128 @@ Similarly to CodeSandbox, Glitch allows you to “remix” the MML Starter Proje
...
+```bash
+sudo certbot --nginx
+```
+
+At this point, you may access your app through: `https://<YOUR_ROOT_DOMAIN>/`
+
+### Summary
++ Provides more compute resources than the sandbox options and should be more stable
++ More flexible and robust infrastructure that will allow you to securely host multiple applications with subdomains and SSL certificates
++ You have full management controll over your VPS
```
Comment (MarcusLongmuir) [inline src/content/guides/mml-starter-project-hosting/body.mdx:237]: Typo: `controll`/`control`

File: src/content/guides/mml-starter-project-hosting/body.mdx [hunk 5]
```diff
@@ -112,3 +112,128 @@ Similarly to CodeSandbox, Glitch allows you to “remix” the MML Starter Proje
...
+`http://<your_static_ip>:8080`
+
+* You may now assign a Root Domain to your instance by clicking in `Networking` on the left panel, going to `Domains` and clicking on `Assign domain` to point a domain name to the static IP you created. To know more details about pointing your domain to a Lightsail Instance, you can check the [official How to guide](https://lightsail.aws.amazon.com/ls/docs/en_us/articles/amazon-lightsail-routing-to-instance) about it on AWS.
+
+* Install Nginx to act as a proxy to pass the connections for your IP address on the default port to 8080, through the following command:
+```bash
+sudo apt install nginx
+```
+
+* Prepare Nginx to proxy your default income connection at port 80 to port 8080 (the port the MML starter project listens to), by creating a configuration file at `/etc/nginx/sites-available/<YOUR_ROOT_DOMAIN>` with your favorite text editor. The content the configuration file should look like this:
```
Comment (MarcusLongmuir) [inline src/content/guides/mml-starter-project-hosting/body.mdx:179]: Backticks for `80` and `8080` to be consistent

File: src/content/guides/mml-starter-project-hosting/body.mdx (hunk 2 above)
Comment (TheCodeTherapy) [reply src/content/guides/mml-starter-project-hosting/body.mdx]: fixed [here](https://github.com/mml-io/mml-website/pull/44/commits/cae96d07c2900ba7476904860c8865aab6d280d9#diff-d736a3355273bc438fefdc857a02e362971869503ca171606443482b8a8cd478R129)

File: src/content/guides/mml-starter-project-hosting/body.mdx (hunk 3 above)
Comment (TheCodeTherapy) [reply src/content/guides/mml-starter-project-hosting/body.mdx]: fixed [here](https://github.com/mml-io/mml-website/pull/44/commits/cae96d07c2900ba7476904860c8865aab6d280d9#diff-d736a3355273bc438fefdc857a02e362971869503ca171606443482b8a8cd478R161)

File: src/content/guides/mml-starter-project-hosting/body.mdx (hunk 4 above)
Comment (TheCodeTherapy) [reply src/content/guides/mml-starter-project-hosting/body.mdx]: fixed [here](https://github.com/mml-io/mml-website/pull/44/commits/cae96d07c2900ba7476904860c8865aab6d280d9#diff-d736a3355273bc438fefdc857a02e362971869503ca171606443482b8a8cd478R237)

File: src/content/guides/mml-starter-project-hosting/body.mdx (hunk 5 above)
Comment (TheCodeTherapy) [reply src/content/guides/mml-starter-project-hosting/body.mdx]: fixed [here](https://github.com/mml-io/mml-website/pull/44/commits/cae96d07c2900ba7476904860c8865aab6d280d9#diff-d736a3355273bc438fefdc857a02e362971869503ca171606443482b8a8cd478R179)

**Pull request merged**